client/post-editor/editor-author/index.jsx

```diff
@@ -15,6 +15,10 @@ var Gravatar = require( 'components/gravatar' ),
 	config = require( 'config' ),
 	stats = require( 'lib/posts/stats' );
 
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { changeAuthor } from 'state/ui/editor/post/actions';
+
 var EditorAuthor = React.createClass( {
```

Comment:
By introducing `import` syntax, this is now an ESLint warning. Consider using `const` instead of `var`.